test/fluree/db/reasoner/owl2rl_test.clj

```diff
@@ -200,40 +213,40 @@
                                              "@type" "?t"}})))
               "ex:brian should be of type ex:Person and ex:Child")
 
-          (is (= ["ex:brian"]
+          (is (= ["ex:brian" "ex:carol"]
                  @(fluree/query db-prp-dom
                                 {:context {"ex" "http://example.org/"}
                                  :select  "?s"
                                  :where   {"@id"   "?s"
                                            "@type" "ex:Child"}}))
-              "ex:brian should be the only subject of type ex:Child")))
+              "ex:brian and ex:carol should be the only subjects of type ex:Child")))
 
       (testing "Testing rdfs:range - rule: prp-rng"
         (let [db-prp-rng @(fluree/reason
                             db-base :owl2rl
                             [{"@context"   {"ex"   "http://example.org/"
                                             "owl"  "http://www.w3.org/2002/07/owl#"
                                             "rdfs" "http://www.w3.org/2000/01/rdf-schema#"}
-                              "@id"        "ex:parent"
+                              "@id"        "ex:parents"
                               "@type"      ["owl:ObjectProperty"]
-                              "rdfs:range" [{"@id" "ex:Person"} {"@id" "ex:Parent"}]}])]
+                              "rdfs:range" [{"@id" "ex:Person"} {"@id" "ex:parents"}]}])]
 
-          (is (= (list "ex:Parent" "ex:Person")
+          (is (= (list "ex:Person" "ex:parents")
```

Comment:
This `list` call is extraneous here 
```clojure
(= (list 1 2 3)
   [1 2 3]
   '(1 2 3))
;;=> true
```

Reply:
Is the result order not stable? My thought is that it should be, but I'm not that familiar with it.

Reply:
I'm not sure, I think this used `list` before my changes. A lot of the existing tests are formatted like this.